pbj-core/gradle/modules.properties

```diff
@@ -1,3 +1,4 @@
 com.github.spotbugs.annotations=com.github.spotbugs:spotbugs-annotations
 com.google.protobuf=com.google.protobuf:protobuf-java
 org.antlr.antlr4.runtime=org.antlr:antlr4-runtime
+com.hedera.node.hapi=com.hedera.hashgraph:hapi
```

Comment:
I'm not an expert in Gradle, so I've no idea if this adds a dependency on HAPI, but if it does and the HAPI happens to want to use PBJ, too, wouldn't this create a circular dependency?

Reply:
It would have been OK in this case, as the project that HAPI depends on is `runtime` and the gradle plugin, but not the Helidon module. But anyway this was wrong, it was leftover from early stages where I was using HAPI as a shortcut to get PBJ objects for testing, but then I switched to protoc objects for testing.

I will switch back to PBJ object, but before I can do that, I need to work on the gradle build. To prevent this PR from losing focus, I decided to just use protoc for now and do a follow on PR to address the gradle build more holistically